templates/scenarios/notification-http-trigger-csharp/infra/azure.bicep

```diff
@@ -68,27 +85,18 @@ resource functionApp 'Microsoft.Web/sites@2021-02-01' = {
       ftpsState: 'FtpsOnly'
     }
   }
-  identity: {
-    type: 'UserAssigned'
-    userAssignedIdentities: {
-      '${userAssignedIdentityId}': {} // The identity is used to access other Azure resources
-    }
-  }
 }
 
-// Azure Storage is required when creating Azure Function instance
-resource storage 'Microsoft.Storage/storageAccounts@2021-06-01' = {
-  name: storageName
-  kind: 'StorageV2'
-  location: resourceGroup().location
-  sku: {
-    name: storageSku // You can follow https://aka.ms/teamsfx-bicep-add-param-tutorial to add functionStorageSku property to provisionParameters to override the default value "Standard_LRS".
+resource functionAppSettings 'Microsoft.Web/sites/config@2021-02-01' = {
+  name: '${functionAppName}/appsettings'
+  properties: {
+    BOT_ID: botAadAppClientId // ID of your bot
+    BOT_PASSWORD: botAadAppClientSecret // Secret of your bot
   }
 }
 
-output sku string = functionAppSKU
-output appName string = functionAppName
-output domain string = functionApp.properties.defaultHostName
-output appServicePlanName string = serverfarmsName
-output functionAppResourceId string = functionApp.id
-output functionEndpoint string = 'https://${functionApp.properties.defaultHostName}'
\ No newline at end of file
+output BOT_DOMAIN string = functionApp.properties.defaultHostName
+output APP_SERVICE_PLAN_NAME string = serverfarmsName
+output FUNCTION_APP_RESOURCE_ID string = functionApp.id
```

Comment:
to keep output key aligned, replace it with BOT_AZURE_FUNCTION_APP_RESOURCE_ID

Reply:
sure